MergerLogic/ImageProcessing/ImageFormatter.cs

```diff
@@ -101,6 +101,22 @@ public static void ConvertToFormat(IMagickImage image, TileFormat format)
             return null;
         }
 
+        public static TileFormat? GetTileFormat(IMagickImage<byte> image) {
+            if(image.IsOpaque) {
+                image.Format = MagickFormat.Jpeg;
+            }
+
+            if (image.Format == MagickFormat.Jpg || image.Format == MagickFormat.Jpeg) {
+                return TileFormat.Jpeg;
+            }
+
+            if (image.Format == MagickFormat.Png || image.Format == MagickFormat.Png32) {
```

Comment:
can we recieve other Png's? (8, 16, 64, etc?)

Reply:
Yes, this allows `png` as you can see. I'm not sure we want to allow other types because of the known problem when upscaling with `mapproxy`.
We can consider only allowing one type of `png` as configuration (converting all of them could impact performance but is possible too).

Reply:
please add test

Reply:
There were already tests for these cases. Added some cases for getting the format of `png` with different bit encoding.